manifests/install/charts/as-a-second-scheduler/values.yaml

```diff
@@ -23,3 +23,27 @@ plugins:
   enabled: ["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch","NodeResourcesAllocatable"]
   disabled: ["PrioritySort"] # only in-tree plugins need to be defined here
 
+pluginConfig:
```

Comment:
Let's add some comments in this section:

```yaml
# Customize the enabled plugins' config.
# Refer to the "pluginConfig" section of manifests/<plugin>/scheduler-config.yaml.
# For example, for Coscheduling plugin, you want to customize the permit waiting timeout to 10 seconds:
- name: Coscheduling
  args:
    permitWaitingTimeSeconds: 10 # default is 60
# Or, customize the other plugins
# - name: NodeResourceTopologyMatch
#   args:
#     scoringStrategy:
#       type: MostAllocated # default is LeastAllocated
```

Reply:
done.

Reply:
@Huang-Wei I've updated this section, do I need to add more configs such as `manifests/<plugin>/scheduler-config-v1beta3.yaml` to support v1beta3?

Reply:
We can leave that as a follow-up.